x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -272,6 +293,7 @@ public boolean equals(Object o) {
                 Objects.equals(indicesOptions, that.indicesOptions) &&
                 Objects.equals(filter, that.filter) &&
                 Objects.equals(timestampField, that.timestampField) &&
+                Objects.equals(tieBreakerField, that.tieBreakerField) &&
                 Objects.equals(eventCategoryField, that.eventCategoryField) &&
                 Objects.equals(implicitJoinKeyField, that.implicitJoinKeyField) &&
                 Objects.equals(searchAfterBuilder, that.searchAfterBuilder) &&
```

Comment:
This needs to be cleaned-up since we don't use it - future work if anybody wants to pick it.

Reply:
Btw, this is about `implictJoinKeyField` not `tieBreakerField`.